FightPandemics/Features/Feed/Card/FeedViewCell.swift

```diff
@@ -28,11 +28,39 @@ import UIKit
 
 class FeedViewCell: UICollectionViewCell {
     static let reuseIdentifier = String(describing: self)
+
+    var characters = ["Link", "Zelda", "Ganondorf", "Midna"]
+
+    // The Offer Request Card (Feed)
+    let offerReqCard = OfferRequestCard(headerData: .init(postedDate: "P", reqOrOffer: "O", supplyType: "M"), subHeaderData: .init(initials: "AM", name: "Ana Muller", location: "Berlin, Germany"), bodyData: .init(tit: "I", message: "E"), footerData: .init(numOfLikes: 4, numOfComments: 7))
+
+    let postCardTableView = UITableView()
```

Comment:
This tableView appears not used, we need to populate this tableView with list of offers, requests, or all as the case maybe. Let me know if you don't understand so that I can explain more.

Reply:
Yes, I understand. 
But I think that this implementation maybe should be in another ticket. What do you think?